GitHubReactions.ps1

```diff
@@ -0,0 +1,493 @@
+
+        Gets a reaction using Get-GitHubReaction and pipes it into Remove-GitHubReaction.
+
+    .NOTES
+        Currently, this only supports reacting to issues and pull requests. Issue comments, commit comments and PR comments will come later.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParameterSetName='Elements',
+        ConfirmImpact='High')]
+    [Alias('Delete-GitHubReaction')]
+    [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSReviewUnusedParameter", "", Justification="One or more parameters (like NoStatus) are only referenced by helper methods which get access to it from the stack via Get-Variable -Scope 1.")]
+    param(
+        [Parameter(
+            Mandatory,
+            ParameterSetName='Elements')]
+        [string] $OwnerName,
+
+        [Parameter(
+            Mandatory,
+            ParameterSetName='Elements')]
+        [string] $RepositoryName,
+
+        [Parameter(
+            Mandatory,
+            ValueFromPipelineByPropertyName,
+            ParameterSetName='Uri')]
+        [Alias('RepositoryUrl')]
+        [string] $Uri,
+
+        [Parameter(Mandatory, ParameterSetName='Elements', ValueFromPipelineByPropertyName)]
+        [Parameter(Mandatory, ParameterSetName='Uri', ValueFromPipelineByPropertyName)]
+        [Alias('Issue')]
+        [Alias('PullRequest')]
+        [Alias('PullRequestNumber')]
+        [int64] $IssueNumber,
+
+        [Parameter(Mandatory, ValueFromPipelineByPropertyName, ValueFromPipeline)]
+        [int64] $ReactionId,
+
+        [Parameter()]
+        [switch] $Force,
+
+        [string] $AccessToken,
+
+        [switch] $NoStatus
+    )
+
+    Write-InvocationLog
+
+    $elements = Resolve-RepositoryElements
+    $OwnerName = $elements.ownerName
+    $RepositoryName = $elements.repositoryName
+
+    $telemetryProperties = @{
+        'OwnerName' = (Get-PiiSafeString -PlainText $OwnerName)
+        'RepositoryName' = (Get-PiiSafeString -PlainText $RepositoryName)
+    }
+
+    $uriFragment = "/repos/$OwnerName/$RepositoryName/issues/$IssueNumber/reactions/$ReactionId"
+    $description = "Removing reaction $ReactionId for Issue $IssueNumber in $RepositoryName"
+
+    if ($Force.IsPresent -or $PSCmdlet.ShouldProcess($ReactionId, "Removing reaction for Issue $IssueNumber in $RepositoryName"))
+    {
+        $params = @{
+            'UriFragment' = $uriFragment
+            'Description' =  $description
+            'Method' = 'Delete'
+            'AcceptHeader' = $script:squirrelGirlAcceptHeader
+            'AccessToken' = $AccessToken
+            'TelemetryEventName' = $MyInvocation.MyCommand.Name
+            'TelemetryProperties' = $telemetryProperties
+            'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+        }
+
+        return Invoke-GHRestMethod @params
+    }
+}
+
+filter Add-GitHubReactionAdditionalProperties
+{
+<#
+    .SYNOPSIS
```

Comment:
The CBH is missing linefeeds between sections, as well as `.PARAMETER` entries for your other parameters.  Please leave `TypeName` as the second parameter, and the other parameters as successive parameters.